### PR TITLE
T1 39.users profile page

### PR DESCRIPTION
**Pull request by diegoplatap**
# Users can update their profile info

- Added profile routes with the auth middleware. 
- Create functionality to let users update their profile data and get storage in the database.

Previous data when the user logs in: 
image

After the user update the profile data: (New name)

image

### CODE REVIEW

File: routers/routes.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ const app = express.Router();
 
 app.get("/me", auth, controllers.loadUser);
 app.post("/login", controllers.login);
+app.get("/:id/profile", auth);
```
Comment (leoneloliveros) [inline routers/routes.js:8]: Controller functionality isn't called?

Comment (diegoplatap) [reply routers/routes.js:8]: Controller is called in : app.put("/:id/profile", auth, controllers.updateProfile) and that happens when the user click's the update profile button. 

In our app.js we have a useEffect which loads the user when a dispatch ocurrs.

Comment (leoneloliveros) [reply routers/routes.js:8]: So this route is not use in the app?

Comment (diegoplatap) [reply routers/routes.js]: Thanks Leonel, you are rigth the app was not using the route. I just commited the change.

**Review by leoneloliveros — APPROVED** [review]

**Pull request merged**